doc/api/cli.md

```diff
@@ -2249,6 +2249,19 @@ Starts the Node.js command line test runner. This flag cannot be combined with
 See the documentation on [running tests from the command line][]
 for more details.
 
+### Environment Variable
```

Comment:
This belongs in the existing environment variables section of the document. It should go immediately after the `NODE_TEST_CONTEXT` entry. It should match the formatting of the other environment variables in the document, and should not have its own header.

Reply:
something like this ?

```md
### `NODE_TEST_CONTEXT=value`

If `value` equals `'child'`, test reporter options will be overridden and test
output will be sent to stdout in the TAP format. If any other value is provided,
Node.js makes no guarantees about the reporter format used or its stability.

### `NODE_TEST_WORKER_ID`

This environment variable is set by the test runner. When using process-level test
isolation, each worker process is assigned a unique ID, starting at `1`. This is
set to 1 for all tests when test isolation is disabled.

Reply:
Yep